Update for ESP32 Core 2.0.9 for Arduino, improved documentation

Murkas94 opened the pull request:
I've updated the files to work with version 2.0.9 of the ESP32 Core for Arduino. I've also improved the documentation on how to install the files and how to build them on your own.

My build actualy created a lot of changed files in comparisson to the files of the official ESP32 Core and i've added them all, because i am actualy not sure, which changes are relevant and which are not.

Murkas94 committed:
Update for ESP32 Core 2.0.9 for Arduino, improved documentation

Xrono32 commented:
> I've updated the files to work with version 2.0.9 of the ESP32 Core for Arduino. I've also improved the documentation on how to install the files and how to build them on your own.
> 
> My build actualy created a lot of changed files in comparisson to the files of the official ESP32 Core and i've added them all, because i am actualy not sure, which changes are relevant and which are not.

Good afternoon! 
How can I download your version of the library?

Murkas94 commented:
> > I've updated the files to work with version 2.0.9 of the ESP32 Core for Arduino. I've also improved the documentation on how to install the files and how to build them on your own.
> > My build actualy created a lot of changed files in comparisson to the files of the official ESP32 Core and i've added them all, because i am actualy not sure, which changes are relevant and which are not.
> 
> Good afternoon! How can I download your version of the library?

Click on my branch/fork ("[Murkas94:version_update/2_0_9](https://github.com/Murkas94/arduino-esp32-solo/tree/version_update/2_0_9)").
There is a green button "Code". Click it, then "download as zip" and there you are.

Xrono32 commented:
> > > I've updated the files to work with version 2.0.9 of the ESP32 Core for Arduino. I've also improved the documentation on how to install the files and how to build them on your own.
> > > My build actualy created a lot of changed files in comparisson to the files of the official ESP32 Core and i've added them all, because i am actualy not sure, which changes are relevant and which are not.
> > 
> > 
> > Good afternoon! How can I download your version of the library?
> 
> Click on my branch/fork ("[Murkas94:version_update/2_0_9](https://github.com/Murkas94/arduino-esp32-solo/tree/version_update/2_0_9)"). There is a green button "Code". Click it, then "download as zip" and there you are.

Thank you! I clicked on the wrong link before (I just started using GitHub).
I downloaded your library, changed all files on Arduino directory C:\Users\100\AppData\Local\Arduino15\packages\esp32\hardware\esp32\2.0.9\tools\sdk\esp32 
The project, originally made for a dual-core ESP-32, compiled and worked as it should!
Thahks!
